src/filters/frame_skipping_pp.py

```diff
@@ -0,0 +1,52 @@
+from src.filters.abstract_pp import AbstractPP
+from src.utils import image_utils
+from typing import List
+
+
+class frameSkippingPP(AbstractPP):
+
+    def __init__(self, threshold=0.0, compare_foreground=False, 
+                 distance_metric='absolute_difference'):
+        self.threshold = threshold
+        self.compare_foreground = compare_foreground
+        self.distance_metric = distance_metric
+
+    def predict(self, batch) -> List[bool]:
```

Comment:
From your logic, parameter 'batch' is of type List[FrameBatch]. 

All our APIs PPs and UDFs take a FrameBatch as input and not List[FrameBatch]. Please change your logic accordingly. 

Use type hints for parameters (code style suggestion)

Reply:
Modified input to pp to be a batch of frames.